R/tidyme.R

```diff
@@ -87,9 +87,16 @@ tidyme.survfit <- function(x, ...) {
       retme[["n.strata"]] <- rep(x[["n"]], x[["strata"]])
     }
   }
-  
+
   attr(retme, "survfit_object") <- survfit_object
+  # modify strata label, removing ref to raw variable name
+  if (!is.null(purrr::pluck(survfit_object, "estimate_KM_args", "strata"))) {
```

Comment:
can we avoid the dependency on purrr? I dont think we are using it somewhere else.

Reply:
gone!